catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -196,6 +196,9 @@ protected Object executeWithSubject() throws Exception {
 
                 entry = zipInputStream.getNextEntry();
             }
+        } catch (Exception e) {
+            LOGGER.error("Exception while importing metacards", e);
```

Comment:
Fixed the log level. I was originally just trying to make visible, So i changed it to just print out top level error message to console and moved the level down to info.